modules/cli/src/main/java/org/apache/ignite/cli/commands/sql/SqlMetaData.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.cli.commands.sql;
+
+/**
+ * Constants for SQL auto-completion.
+ */
+
+public class SqlMetaData {
+
+    /**
+     * SQL reserved keywords.
+     */
+    public static String[] KEYWORDS = {
```

Comment:
Do we have any plans to share the real actual list of keywords from the ignite sql module? It looks like we will have many custom keywords about data layouts and etc.